rfcs/0022-transaction-structure/0022-transaction-structure.md

```diff
@@ -402,3 +402,78 @@ Cell data hash is just `ckbhash(data)`.
 
 Script hash is `ckbhash(molecule_encode(script))` where `molecule_encode` turns the script structure into a block of binary via molecule. See the definition `Script` in the [schema file](https://github.com/nervosnetwork/ckb/blob/a6733e6af5bb0da7e34fb99ddf98b03054fa9d4a/util/types/schemas/blockchain.mol#L28-L32).
 
+## Appendix B: Syscalls
+
+| VM Ver. | Syscall ID | C Function Name | Description | Return Value | Partial Loading[^2] | 1st Arg | 2nd Arg | 3rd Arg | 4th Arg | 5th Arg | 6th Arg | C Example |
```

Comment:
There are too many columns, making the viewing experience on GitHub really bad.

My suggestion is removing most columns and add links to their detailed documentations.

```
| VM Ver. | Syscall ID | C Function Name | Description |
| --- | --- | --- | --- |
| 1 | 93 | [ckb_exit][] | terminate execution with the specified return code |

[ckb_exit]: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0009-vm-syscalls/0009-vm-syscalls.md#exit
```

which gives us:

| VM Ver. | Syscall ID | C Function Name | Description |
| --- | --- | --- | --- |
| 1 | 93 | [ckb_exit][] | terminate execution with the specified return code |

[ckb_exit]: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0009-vm-syscalls/0009-vm-syscalls.md#exit

Reply:
Drafting new RFC based on the comments. One question, is the new RFC solely about vm syscalls? Or, _Appendix C: Constants_ will still be included?